packages/astro/src/core/create-vite.ts

```diff
@@ -309,6 +310,6 @@ function isCommonNotAstro(dep: string): boolean {
 	);
 }
 
-function stringifyForDefine(value: string | undefined): string {
+function stringifyForDefine(value: string | undefined | object): string {
```

Comment:
Why was this changed?

Reply:
<img width="960" alt="image" src="https://github.com/withastro/astro/assets/26745086/3ff42772-dacb-4e06-92b7-fa7d396093b5">
Here stringify `assetsPrefix`